db/migrate/20170628104617_add_columns_to_conferences.rb

```diff
@@ -0,0 +1,9 @@
+class AddColumnsToConferences < ActiveRecord::Migration[5.1]
+  def change
+    add_column :conferences, :gid, :integer
```

Comment:
Nevermind, for some reason I completely overlooked what you've written in this PRs description 🙈